src/System.Management.Automation/engine/SessionStateProviderAPIs.cs

```diff
@@ -1749,7 +1468,7 @@ internal int ProviderCount
                 return count;
             }
         } // ProviderCount
-    } // SessionStateInternal class
+    } // SessionStateInternal class}
```

Comment:
This looks strange.  Can you remove the bracket from the comment?

Reply:
Yes, that was probably just a mistake